stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -1,10 +1,11 @@
 // RUN: stablehlo-opt --mlir-print-op-generic %s.bc | FileCheck %s
 // RUN: stablehlo-translate --deserialize %s.bc | stablehlo-translate --serialize --target=0.10.0 | stablehlo-opt --mlir-print-op-generic | FileCheck %s
-// RUN: stablehlo-translate --deserialize %s.bc | stablehlo-opt > %t.0
-// RUN: stablehlo-opt --strip-debuginfo %s > %t.1
-// RUN: diff %t.0 %t.1
-// RUN: stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo %s > %t.2
-// RUN: diff %s.bc %t.2
+// RUN: diff <(stablehlo-translate --deserialize %s.bc | stablehlo-opt) <(stablehlo-opt --strip-debuginfo %s)
+// RUN: diff %s.bc <(stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo %s)
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo -emit-bytecode -debug-only=vhlo-bytecode %s 2>&1 | FileCheck --check-prefix=CHECK-WARN %s
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo -emit-bytecode %s | stablehlo-opt -debug-only=vhlo-bytecode 2>&1 | FileCheck --check-prefix=CHECK-WARN %s
```

Comment:
Not sure I get this: what are we checking with the log output?

Reply:
Making sure we don't hit `Not Implemented` while reading/writing the bytecode.

Reply:
See `stablehlo_legalize_to_vhlo.mlir` for reference. Some of the versioned tests have the `CHECK-WARN-NOT`, but they don't actually run any commands that might check for these. IMO we should check for these in all of the tests.

Reply:
+1. I think I reviewed this initial change, bit didn't fully understand how filecheck worked at that time. I think we do need this.